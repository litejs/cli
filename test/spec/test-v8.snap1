v8.js
  ✔ 1. it should test optimization [3/3]
  ✔ 2. it should test fast properties [2/2]
1..2
# pass  2/2 [5/5]
