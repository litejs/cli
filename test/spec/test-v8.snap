v8.js
  ✔ it should test optimization #1 [0/0]
  ✔ it should test fast properties #2 [0/0]
1..2
# pass  2/2 [0/0]
